components/tikv_util/src/log.rs

```diff
@@ -83,6 +83,18 @@ macro_rules! trace(($($args:tt)+) => {
     ::slog_global::trace!($($args)+)
 };);
 
+/// Logs a infor or debug level message using the slog global logger.
```

Comment:
```suggestion
/// Logging an info or debug level message using the slog global logger.
```

Reply:
It keeps consistent with other log functions